Windows CI is broken

See https://github.com/kiwix/kiwix-tools/actions/runs/12434279020/job/34717585726?pr=719

```
libxapian.a(matcher_synonympostlist.cc.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in kiwix-manage.cpp.obj
libxapian.a(matcher_selectpostlist.cc.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in kiwix-manage.cpp.obj
libxapian.a(matcher_selectpostlist.cc.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in kiwix-manage.cpp.obj
libdocopt.a(docopt.cpp.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in kiwix-manage.cpp.obj
libdocopt.a(docopt.cpp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in kiwix-manage.cpp.obj

   Creating library src\manager\kiwix-manage.lib and object src\manager\kiwix-manage.exp

LINK : warning LNK4098: defaultlib 'MSVCRT' conflicts with use of other libs; use /NODEFAULTLIB:library

src\manager\kiwix-manage.exe : fatal error LNK1319: 404 mismatches detected
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -41,7 +41,7 @@ jobs:
         run: |
           set PKG_CONFIG_PATH=%cd%\BUILD_win-amd64\INSTALL\lib\pkgconfig
           set CPPFLAGS=-I%cd%\BUILD_win-amd64\INSTALL\include
-          meson.exe setup . build -Dstatic-linkage=true --buildtype=debug
+          meson.exe setup . build -Dstatic-linkage=true --buildtype=release
           cd build
           ninja.exe
 

